ci/run-tests.py

```diff
@@ -68,7 +68,16 @@ def change_setting(content, setting, value):
 if args.headless:
     init_contents = change_setting(init_contents, 'PRINT_MODE', 'TEXT')
 
-test_init_file = 'dfhackzzz_test.init'  # Core sorts these alphabetically
+init_path = 'dfhack-config/init'
```

Comment:
Did this change make it into the buildmaster image? I think that should happen before merging this PR. CC @BenLubar

Reply:
It's easiest if it's merged first - the build script for the container pulls the script from the develop branch at build time.

Reply:
Merged

Reply:
Rebuilding the containers...

Reply:
Should be good to go now.